src/Cart.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { connect } from 'react-redux';
+import Modal from './UI/Modal';
+import './Cart.css';
+import { BsFillTrashFill } from "react-icons/bs";
+
+const Cart = (props) => {
+
+    const [ordering, setOrdering] = useState(false);
+    const [text, setText] = useState();
+
+    const orderCart = () => {
+        let order = props.cart.map(product => {
+            return {
+                id: product.id,
+                amount: product.amount
+            }
+        });
+        setOrdering(true)
+        axios.post('http://localhost:8080/order', {products: order})
+        .then(response => {
+            const data = response.data;
+            setText('Successfully placed the order id:' + data.id + ' total price: ' + data.totalPrice);
+            props.onClearCart();
+            
+        })
+        .catch(err => {
+            setText('An error has occured, please retry ordering!');
+            
+        });
+    }
+
+    const onModal = () => {
+        setOrdering(false);
+    }
+    
+    return (
+        <div>
+                <Modal show={ordering && text} onOk={onModal}>
+                    {text}
+                </Modal>
+        <div className="card">
+            <h5 className="card-header">Shipping Cart</h5>
+            <div className="card-body">
+                {props.cart.map(crt => {
+                    return (
+                        <div className="row">
```

Comment:
Also here and in col, you can use react-bootstrap components instead

Reply:
In every place in your code you did use in div with classname for bootstrap style, check if you can do use in react-bootstrap component instead
You can read about react-bootstrap components here: https://react-bootstrap.github.io